FarihaIS/VanRoomies
# bugfix(listings) FE - Listings Bugfix Mega PR

# Welcome to VanRoomies! 👋✈️

## Description of the change:
- Rental Price is now shown in listings
- Rental Price can be edited within the listing (and will be validated before being sent to the BE)
- The default zoom for the map view has been increased to better capture the app's scope of Vancouver
- Added check for title length when editing listings
- Rewrote manageListingsTest to reflect the new changes and better line up with test specifications

## How to manually test:
1. Start MongoDB and the server
2. Run the test suite
3. Go to ListingsFragment
4. Create a new listing
5. Observe the rental price and try editing it. It should be numerical. Invalid values will cause a Toast message to pop up
6. Check the View Map button and note the default zoom
7. Try editing a listing title to a title with < 5 characters. A Toast message should pop up.
